src/TestExceptions.java

```diff
@@ -0,0 +1,27 @@
+import Exceptions.WrongLoginException;
+import Exceptions.WrongPasswordException;
+
+import java.util.regex.Pattern;
+
+public class TestExceptions {
+
+    private static final Pattern LOGIN_CHECK_SYMBOLS = Pattern.compile("[a-zA-Z0-9_]{1,20}");
+    private static final Pattern PASSWORD_CHECK_SYMBOLS = Pattern.compile("[a-zA-Z0-9_]{1,20}");
+
+    public static void enteringPersonalData(String login, String password, String confirmPassword) {
+
+        if (!LOGIN_CHECK_SYMBOLS.matcher(login).matches()) {
```

Comment:
```suggestion
        if (!login.matches("\\w{1,20}")) {
```
или так, без создания матчера и паттерна